components/cloud/azure/Cargo.toml

```diff
@@ -6,20 +6,25 @@ publish = false
 
 [dependencies]
 async-trait = "0.1"
-azure_core = { version = "0.1.0", git = "https://github.com/Azure/azure-sdk-for-rust" }
-azure_identity = { version = "0.1.0", git = "https://github.com/Azure/azure-sdk-for-rust" }
-azure_storage = { version = "0.1.0", git = "https://github.com/Azure/azure-sdk-for-rust", default-features = false, features = ["account", "blob"] }
```

Comment:
why remove "default-features = false features = ["account", "blob"]"

Reply:
About the first question:
> why remove "default-features = false features = ["account", "blob"]".

In the latest version, `account` feature is already abandoned, related traits have been included in `azure_storage` and `azure_storage_blob`. The `default-features = false` should not be ignored and added now.

And for:
> Also why use the same dependent component with different names?

They are different libs released by `azure-sdk-for-rust` and maintained in the same repo.